network/endpoint.go

```diff
@@ -94,7 +99,22 @@ type EndpointInfo struct {
 	SkipDefaultRoutes        bool
 	HNSEndpointID            string
 	HNSNetworkID             string
-	HostIfName               string
+	HostIfName               string // unused in windows, and in linux
+	// Fields related to the network are below
+	MasterIfName                  string
+	AdapterName                   string
+	NetworkID                     string
+	Mode                          string
+	Subnets                       []SubnetInfo
+	PodSubnet                     SubnetInfo
```

Comment:
Whats the diff between subnets and podsubnet?

Reply:
PodSubnet does not seem to be used in linux or windows-- deleting from endpoint info struct